liota/entities/registered_entity.py

```diff
@@ -36,6 +36,16 @@ def __init__(self, ref_entity, ref_dcc, reg_entity_id):
         self.ref_entity = ref_entity
         self.ref_dcc = ref_dcc
         self.reg_entity_id = reg_entity_id
+        self.parent = None
```

Comment:
Okay, this is reasonable. 

entity.entity_type, entity.name (in logs) is the only place where the parent object is required in the set_properties method. We can actually pass entity_type as an attribute like we set the reg_entity_id during create_relationship call. In future, when we register Metric as a separate Entity we won`t require this code as property would be set in the similar way they are set on the device or gateway. 

Also, AWS DCC if they want to set attribute(property) they will set them directly on the metric object rather then on the parent.